packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -155,8 +152,11 @@ function extractEvents(
     targetContainer,
   );
   const shouldProcessPolyfillPlugins =
-    (eventSystemFlags & IS_CAPTURE_PHASE) === 0 ||
-    capturePhaseEvents.has(topLevelType);
+    (eventSystemFlags & IS_EVENT_HANDLE_NON_MANAGED_NODE) === 0 &&
+    (eventSystemFlags & IS_NON_DELEGATED) === 0 &&
+    ((eventSystemFlags & IS_CAPTURE_PHASE) === 0 ||
+      topLevelType === TOP_FOCUS ||
```

Comment:
This needs to be rebased on https://github.com/facebook/react/pull/19186 and changed?